serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc

```diff
@@ -1,107 +1,584 @@
+----
+
 [NOTE]
 ====
-Data index service and the support for jobs information will be available in future releases.
+This is the recommended approach when you execute the Job Service in kubernetes.
+The timeouts showcase example xref:use-cases/timeout-showcase-example.adoc#execute-quarkus-project-standalone-services[Quarkus Workflow Project with standalone services] contains an example of this configuration, https://github.com/kiegroup/kogito-examples/blob/main/serverless-workflow-examples/serverless-workflow-timeouts-showcase-extended/kubernetes/jobs-service-postgresql.yml#L65[see].
 ====
 
-=== Job Service persistence
+[#using-java-like-system-properties]
+=== Using system properties with java like names
+
+To configure the image by using system properties you must pass one property per parameter, however, in this case, all these properties are passed as part of a single environment with the name `JAVA_OPTIONS`.
+
+[source, bash]
+----
+docker run -it -e JOBS_SERVICE_PERSISTENCE=postgresql -e JAVA_OPTIONS='-Dmy.sys.prop1=value1 -Dmy.sys.prop2=value2' \
+quay.io/kiegroup/kogito-jobs-service-allinone:latest
+----
+
+[NOTE]
+====
+I case that you need to convert a java like property name, to the corresponding environment variable name, to use the environment variables configuration alternative, you must apply the naming convention defined in the link:{quarkus_config_reference_url}#environment-variables[Quarkus Configuration Reference].
+For example, the name `quarkus.datasource.jdbc.url` must be converted to `QUARKUS_DATASOURCE_JDBC_URL`.
+====
+
+[#job-service-global-configurations]
+== Global configurations
+
+Global configurations that affects the job execution retries, startup procedure, etc.
+
+[tabs]
+====
+Using environment variables::
++
+
+[cols="2,1,1"]
+|===
+|Name |Description |Default
+
+|`KOGITO_JOBS_SERVICE_BACKOFFRETRYMILLIS`
+|A long value that defines the retry back-off time in milliseconds between job execution attempts, in case the execution fails.
+|`1000`
+
+|`KOGITO_JOBS_SERVICE_MAXINTERVALLIMITTORETRYMILLIS`
+|A long value that defines the maximum interval in milliseconds when retrying to execute jobs, in case the execution fails.
+|`60000`
+
+|===
 
-An important configuration aspect of the Job Service is the persistence mechanism, where all job information is stored in a database that makes this information durable upon service restarts and guarantees no information is lost.
+Using system properties with java like names::
++
+
+[cols="2,1,1"]
+|===
+|Name |Description |Default
+
+|`kogito.jobs-service.backoffRetryMillis`
+|A long value that defines the retry back-off time in milliseconds between job execution attempts, in case the execution fails.
+|`1000`
+
+|`kogito.jobs-service.maxIntervalLimitToRetryMillis`
+|A long value that defines the maximum interval in milliseconds when retrying to execute jobs, in case the execution fails.
+|`60000`
+
+|===
+
+====
+
+[#job-service-persistence]
+== Persistence
```

Comment:
Maybe we should specify that if size matters, it exists specific persistence images instead of the all-in-one, for example `kogito-jobs-service-infinispan` or `kogito-jobs-service-postgresql`

Reply:
@radtriste Individual images where referenced here as suggested
https://deploy-preview-408--tangerine-crostata-04a6bf.netlify.app/serverlessworkflow/latest/job-services/core-concepts.html#job-service-persistence